GOVERNANCE.md

```diff
@@ -0,0 +1,136 @@
+* How this role is recognized: credited via role="cre" in DESCRIPTION, so they appear as Maintainer on CRAN. 
+
+## Other roles
+
+Each of the below roles is important, and a wiki page should be
+created to explain (1) details of responsibilities, (2) who to contact
+to obtain this role. 
+
+* Triage manager (to decide which issues/PRs to include in next releases), no special permissions.
+* Translation manager (to communicate with translators, and source code changes related to i18n, for example making sure messages are translation-ready, updating the .pot template), no special permissions (updates should happen in a pull request as usual).
+* Performance testing manager (to prevent performance regressions), no special permissions (updates should happen in a pull request as usual).
+* Continuous integration/deployment manager (to maintain script which run tests and build binaries of development branches for user testing before release), no special permissions (updates should happen in a pull request as usual).
+* Reverse dependency manager (to ensure compatibility with other CRAN packages), need access to a computer that can run revdep checks in parallel.
+
+# Decision-making processes
+
+## Definition of Consensus
+
+Most decisions in the project happen by Consensus, which means that no active people (typically Reviewers and/or Committers) have expressed major blocking concerns, in a public discussion (typically in a GitHub issue or pull request). In Consensus, non-response by inactive members indicates tacit agreement.
+
+## Pull Requests
+
+A pull request can be merged as long as there is one approving review (ideally from a reviewer of the affected files, different person from author of PR), and Consensus from active Reviewers and Committers.
+
+## CRAN updates
+
+* Regular CRAN releases should ideally occur twice per year, and can include new features.
+* A hotfix/patch CRAN release should occur when CRAN asks for one, at which time the CRAN maintainer should post an issue on github, and ask others to help fix/prepare the release.
+* Both kinds of releases should be discussed in an issue, and the release should happen only if there is Consensus among active Reviewers and Committers.
+* It is the responsibility of the CRAN maintainer to ensure quality prior to release. This includes CRAN checks, unit tests, performance tests, etc, and these tasks can be delegated to others. 
+
+## Changing this GOVERNANCE.md document
+
+There is no special process for changing this document (submit a PR
+and ask for review).
+
+# Code of conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
```

Comment:
Shouldn't contributors be enough, since some roles are sublevel inclusive?
Otherwise we might need to mention the other roles too.